lib/oli_web/live/sections/open_free_settings.ex

```diff
@@ -36,6 +36,11 @@ defmodule OliWeb.Sections.OpenFreeSettings do
         <Label class="form-check-label"/>
       </Field>
 
+      <Field name={:confirm_students_on_creation} class="mt-2 form-check">
```

Comment:
I find this field name a bit ambiguous, since it could be understood as a check meaning, _"if `confirm_students_on_creation` is true, the users will have to confirm their account on creation"_. 

And it could be also understood as _"when creating a user account, the user is automatically confirmed",_ which I think is what you're coding here.

What do you think about a more explicit field like `omit_students_email_verification_on_creation` or maybe something shorter 😅?

Reply:
Haha you have the same doubt as me while coding it 😄 

I went for `omit_students_email_verification`, let me know if that's better

Reply:
what about `skip_email_verification`? Seems to explain it, simpler and seems like re-use would be higher.